index.src.html

```diff
@@ -2523,9 +2513,9 @@
 };
```

Comment:
We propose deleting `plane.readInto()`, as it is now replaced by `frame.readInto()`. 

The original `frame.planes[i].readInto()` was much less efficient in the common case of needing to read all planes (for mapped YUV frames, this required us to map/unmap 3 times!) and much more verbose to do common things like read out just the visible region (no padding). 

The only missing feature of the new path is the ability to read back just one plane if you desire that (e.g. say you just want Y of YUV to run some ML analysis like face tracking). This can be easily addressed by adding a `discard` field to the layout dict...so the YUV case becomes:

```
let options = {layout: [{},                // y plane, default layout
                        {discard: true},   // u plane, discarded
                        {discard: true}]}; // v plane, discarded
frame.readInto(buff, options);
```

Reply:
Furhtermore, with `planes.readInto()` deleted, we propose deleting `frame.planes` entirely. It's remaining attributes only served as metadata for sizing a BufferSource to provide to `plane.readInto()`. This metadata is no longer needed with the simpler regions-based `frame.readInto()`. Folks can instead determine the minimum size for their destination buffer using using `frame.allocationSize()`.

Reply:
Sorry, lets discuss this in the issue #157 for better visibility.